en/FlyView/FlyView.md

```diff
@@ -1,61 +1,81 @@
 # Fly View
 
-![](../../assets/quickstart/ConnectedVehicle.jpg)
+The Fly View is the main view you will use while flying your vehicle. 
+You can switch between a map view and a video view (if available).
+
+![Fly View](../../assets/fly/fly_view_overview.jpg)
 
-The Fly View is the main view you will use while flying your vehicle. You can switch between a map view and a video view (if available).
 
 ## Map
 
-The map will show you the positions of all the vehicles you are connected to. It will also show you the mission for the current vehicle.
+The map shows the positions of all connected vehicles and the  mission for the current vehicle.
+
+You can drag the map to move it around (the map automatically recentres after a certain amount of time).
+
+Once flying, you can click on the map to set a GoTo position.
+
+
+## Fly Tools {#fly_tools}
 
-## Fly Tools
-On the left edge of the screen you will see the Fly Tools. The order of tools from top to bottom is:
+On the left edge of the screen you will see the *Fly Tools*.
+You can use these to takeoff, return, pause, or continue/start an action (e.g. a mission).
+
+The order of tools from top to bottom is:
 
 * Takeoff/Land
 * Return To Launch
 * Pause
 * Action
 
-### Center Map
-The Center Map tool allow you to center the map around various points such as home position, vehicle and so forth.
-
-### Map Type
-This tool allows you to change the current map type between Street, Satellite and Hybrid (Street+Satellite). The default map provider is Bing since it seems to provide better Hybrid maps. You can change the map provider from the General page of Settings.
-
 ## Video
-At the lower left of the display you will see video output. QGroundControl supports RTP and RTSP video streaming over your vehicles UDP connection. It also support directly connected UVC device support. More details on QGC Video support can be found on the [Video README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md).
 
-By clicking on the video you can make it be the main display for the Fly view.
+At the lower left of the display you will see video output.
```

Comment:
Is any of this section true, and if not, what is the story? I don't have video working.

Reply:
Yes it is true. Will update image.